Dev environment documentation: no instructions to run 'make install'

The [development environment documentation](https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#environment-setup) doesn't state that several extra tools are required:

- [mockgen](https://github.com/golang/mock)
- [ko](https://github.com/google/ko)

On a pristine git checkout of the project with go 1.17 and docker v20.10.12 installed, trying to run `make build` shows:

```
scorecard % make build
# Generating MockCIIClient
mockgen -source=clients/vulnerabilities.go -destination=clients/mockclients/vulnerabilities.go -package=mockrepo -copyright_file=clients/mockclients/license.txt
make: mockgen: No such file or directory
make: *** [checks/mockclients/vulnerabilities.go] Error 1
```

After installing mockgen, compilation progresses until we hit another error:

```
# Run go build on the update script
cd cron/data/update && CGO_ENABLED=0 go build -trimpath -a -tags netgo -ldflags '-X github.com/ossf/scorecard/v4/pkg.gitVersion=v4.0.1-53-g86d8281 -X github.com/ossf/scorecard/v4/pkg.gitCommit=86d8281031f4a55cc0425773863631d072669244 -X github.com/ossf/scorecard/v4/pkg.gitTreeState=clean -X github.com/ossf/scorecard/v4/pkg.buildDate=1643759415 -w -extldflags "-static"'  -o projects-update
go: downloading github.com/go-git/go-git/v5 v5.4.2
go: downloading golang.org/x/tools v0.1.9
go: downloading github.com/go-git/go-billy/v5 v5.3.1
go: downloading github.com/ProtonMail/go-crypto v0.0.0-20210428141323-04723f9f07d7
go: downloading github.com/imdario/mergo v0.3.12
go: downloading github.com/jbenet/go-context v0.0.0-20150711004518-d14ea06fba99
go: downloading github.com/go-git/gcfg v1.5.0
go: downloading github.com/emirpasic/gods v1.12.0
go: downloading github.com/kevinburke/ssh_config v0.0.0-20201106050909-4977a11b4351
go: downloading github.com/xanzy/ssh-agent v0.3.0
go: downloading gopkg.in/warnings.v0 v0.1.2
KO_DATA_DATE_EPOCH=1643759415 KO_DOCKER_REPO=/scorecard LDFLAGS="-X github.com/ossf/scorecard/v4/pkg.gitVersion=v4.0.1-53-g86d8281 -X github.com/ossf/scorecard/v4/pkg.gitCommit=86d8281031f4a55cc0425773863631d072669244 -X github.com/ossf/scorecard/v4/pkg.gitTreeState=clean -X github.com/ossf/scorecard/v4/pkg.buildDate=1643759415 -w -extldflags "-static"" \
        ko publish -B \
                           --push=false \
                           --platform="linux/amd64,linux/arm64,linux/386,linux/arm"\
                           --tags latest,v4.0.1-53-g86d8281,86d8281031f4a55cc0425773863631d072669244 github.com/ossf/scorecard/v4
/bin/bash: ko: command not found
make: *** [scorecard-ko] Error 127
```

Only after installing the tools manually did I discover there is a `make install` target that would install the tooling. Can the documentation be updated to include a `make install` step, or perhaps the makefile can be updated to have `build` depend on the `install` target.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -71,6 +71,7 @@ git clone git@github.com:<user>/scorecard.git $GOPATH/src/github.com/<user>/scor
 
 1.  Ensure you activate module support before continue (`$ export
     GO111MODULE=on`)
+1.  Install the build tools for the project by running the command `make install`
 1.  Run the command `make build` to build the source code
 
 ## PR Process

